assignments/arrays.js

```diff
@@ -62,33 +66,79 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
-let lastCar = 0;
-console.log();
+// The dealer needs the information on the last car in their inventory.  
+//What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
+let lastCar = inventory.length - 1;
+console.log(inventory[lastCar]);
 
 // ==== Challenge 3 ====
-// The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+// The marketing team wants the car models listed alphabetically on the website. 
+//Sort all the car model names into alphabetical order and log the results in the console
+
+let newArray = []
+for (let i = 0; i < inventory.length; i++) {
+
+  newArray.push(inventory[i].car_model)
+}
+
+console.log(newArray.sort());
+
 
 // ==== Challenge 4 ====
-// The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
+// The accounting team needs all the years from every car on the lot. 
+//Create a new array from the dealer data containing only the car years and log the result in the console.
+
 let carYears = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i++) {
+  carYears.push(inventory[i].car_year)
+}
+
+console.log(carYears);
+
 
 // ==== Challenge 5 ====
-// The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
+// The car lot manager needs to find out how many cars are older than the year 2000. 
+//Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
+
 let oldCars = [];
-console.log(); 
+console.log(carYears[5])
+
+for (let i = 0; i < carYears.length; i++) {
+  if (carYears[i] < 2000) {
+  oldCars.push(carYears[i])
+  }
+}
+
+console.log(oldCars.length); 
 
 // ==== Challenge 6 ====
-// A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
+// A buyer is interested in seeing only BMW and Audi cars within the inventory.  
+//Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
+
 let BMWAndAudi = [];
-console.log();
 
+for (let i = 0; i < inventory.length; i++)
+
+  if (inventory[i].car_make === "BMW" || inventory[i].car_make === "Audi")
+
+  BMWAndAudi.push(inventory[i])
```

Comment:
Great use of the `or` Logical Operator `||` here.

Don't forget to open and close `{}` after your IF conditional. Inside of that block of code is what will be executed if the if conditions are met.